_posts/2020-08-15-Named-Graphs.md

```diff
@@ -0,0 +1,539 @@
+sources and exist in their own right.  This situation is very messy
+from a data modelling point of view.  There are competing constraints
+which need to be balanced when attempting to translate the situation
+as it stands within the MS tradition to the digital.  On the one hand,
+there is a solid core of information contained within the major MS
+genealogical collections.  On the other hand, there are substantive
+differences which must be respected.  Additionally, for users of
+IrishGen, it is equally important to be able to search one branch of
+the tradition or the other.  Without Named Graphs, as will be
+discussed shortly, users who only wished to search, for instance, The
+[Book of Leinster](https://en.wikipedia.org/wiki/Book_of_Leinster)
+would have to contort their searches to narrow them.
+
+When using a database such as IrishGen which replicates in
+computational form data which is taken directly from primary sources,
+it is imperative that IrishGen is clear concerning whence that
+information derives.  As it stands, a triple does not carry with it a
+method for partitioning information.  Within IrishGen, there is an
+ad-hoc attempt to this by encoding the MS that a triple is from into
+the triple's URL.  While this method can work with some extra
+diligence from the user, it is far from perfect.  Thankfully, others
+have encountered the same problem and the Semantic Web community has
+created the concept of a "Named Graph".
+
+To illustrate this, the situation in which there is no Named Graph for
+a triple will be shown then the situation with a Named Graph will be
+shown.
+
+```sparql
+prefix irishRel: <http://example.com/earlyIrishRelationship.ttl#> 
+
+select ?a
+where {
+    ?a irishRel:nomName "Find"
+}
+```
+
+The above query in a situation where Named Graphs are not in use will
+return every single triple that happens to match the Basic Graph
+Pattern without respect to what manuscript the triple may have come
+from.  From a research point of view, this is very unhelpful.
+Usually, researchers will want to explore a specific genealogical
+text.  While it can be very interesting to do a meta-analysis of the
+entire corpus as a whole, such as when counting the number of distinct
+individuals recorded, or useful when the user is searching to see if
+information happens to be contained in the genealogies, such as when
+the user has a name from another source and wishes to know if the same
+name appears in the genealogical corpus, a database wide search is
+often not what is wanted.
+
+When a Named Graph is attached to a triple, it is transformed into a
+"quad".  This extra bit of information is attached to every triple in
+a Named Graph.  For instance, a triple will normally look like this to
+a Triplestore:
+
+```turtle
+<http://example.com/LL/lagin.trig#Find> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://xmlns.com/foaf/0.1/Person>
+```
+
+This triple states that "Find is a person".  When the Named Graph is
```

Comment:
Getting really pedantic, it states that "this particular instance of a particular Find is a person". But you'll know better as to whether it is worth making this distinction?